tests/NHSD.BuyingCatalouge.Ordering.Api.Testing.Data/Entities/OrderItemEntity.cs

```diff
@@ -0,0 +1,120 @@
+
+        public TimeUnit? TimeUnit { get; set; }
+
+        public string CurrencyCode { get; set; }
+
+        public int Quantity { get; set; }
+
+        public TimeUnit? EstimationPeriod { get; set; }
+
+        public DateTime? DeliveryDate { get; set; }
+
+        public decimal? Price { get; set; }
+
+        public DateTime Created { get; set; }
+
+        public DateTime LastUpdated { get; set; }
+
+        protected override string InsertSql => @"
+            INSERT INTO dbo.OrderItem
+            (
+                OrderId,
+                CatalogueItemId,
+                CatalogueItemTypeId,
+                CatalogueItemName,
+                OdsCode,
+                ProvisioningTypeId,
+                CataloguePriceTypeId,
+                PricingUnitTierName,
+                PricingUnitDescription,
+                TimeUnitId,
+                CurrencyCode,
+                Quantity,
+                EstimationPeriodId,
+                DeliveryDate,
+                Price,
+                Created,
+                LastUpdated
+            )
+            VALUES
+            (
+                @OrderId,
+                @CatalogueItemId,
+                @CatalogueItemType,
+                @CatalogueItemName,
+                @OdsCode,
+                @ProvisioningType,
+                @CataloguePriceType,
+                @PricingUnitTierName,
+                @PricingUnitDescription,
+                @TimeUnit,
+                @CurrencyCode,
+                @Quantity,
+                @EstimationPeriod,
+                @DeliveryDate,
+                @Price,
+                @Created,
+                @LastUpdated
+            );";
+
+        public static async Task<IEnumerable<OrderItemEntity>> FetchAllAsync(string connectionString)
+        {
+            return await SqlRunner.QueryAsync<OrderItemEntity>(connectionString, @"
+                SELECT  OrderItemId,
+                        OrderId,
+                        CatalogueItemId,
+                        CatalogueItemTypeId AS CatalogueItemType,
+                        CatalogueItemName,
+                        OdsCode,
+                        ProvisioningTypeId AS ProvisioningType,
+                        CataloguePriceTypeId AS CataloguePriceType,
+                        PricingUnitTierName,
+                        PricingUnitDescription,
+                        TimeUnitId AS TimeUnit,
+                        CurrencyCode,
+                        Quantity,
+                        EstimationPeriodId AS EstimationPeriod,
+                        DeliveryDate,
+                        Price,
+                        Created,
+                        LastUpdated
+                FROM    dbo.OrderItem;");
+        }
+
+        public static async Task<OrderItemEntity> FetchByOrderItemId(string connectionString, int orderItemId) => 
+            (await FetchAllAsync(connectionString)).Single(item => orderItemId == item.OrderItemId);
```

Comment:
I'm guessing you've added this because it may be required soon?

Reply:
Yeah just included it as part of the boilerplate. I think I've already used it as part of the update order item endpoint. So will leave it in for now. If I don't use it in the next PR will remove it.